include/small/LeakyReLULayer.hpp

```diff
@@ -40,6 +40,10 @@ class LeakyReLULayer : public Layer<BufferT>
                   << "x" << output_shape[WIDTH]
                   << ")" << std::endl;
 #endif
+
+    BufferT packed_filters(1);
```

Comment:
You can construct m_packed_filters(1) directly in the constructor's initialization list...and set the [0] element of the member variable here.